WhateverGreen/kern_igfx.cpp

```diff
@@ -198,6 +199,18 @@ void IGFX::processKernel(KernelPatcher &patcher, DeviceInfo *info) {
 		KernelPatcher::RouteRequest request("__ZN9IOService20copyExistingServicesEP12OSDictionaryjj", wrapCopyExistingServices, orgCopyExistingServices);
 		patcher.routeMultiple(KernelPatcher::KernelID, &request, 1);
 	}
+	
+	// Enable CFL backlight patch if CPU gen is CFL and model is laptop
+	cflBacklightPatch = (cpuGeneration == CPUInfo::CpuGeneration::CoffeeLake && WIOKit::getComputerModel() == WIOKit::ComputerModel::ComputerLaptop);
+	cflBacklightPatch = cflBacklightPatch || (info->videoBuiltin && info->videoBuiltin->getProperty("enable-cflbklt"));
+	
+	char igfxcflbklt[128];
+	if (PE_parse_boot_argn("igfxcflbklt", igfxcflbklt, sizeof(igfxcflbklt))) {
+		if (strstr(igfxcflbklt, "none"))
```

Comment:
I think this should be `strcmp(…) == 0`, as we should go for an exact match rather than any string, plus I believe any passed value but `force` should probably be considered equivalent to `cflBacklightPatch = false;` for simplicity?